packages/experimental/NativeButton/FluentUIReactNativeButtonMac.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = 'FluentUI-React-Native-Button-Mac'
```

Comment:
Could we just have one podspec. and specify different source files for osx or ios?

Reply:
Good suggestion, I like that we can update the versions in one file.